docs/database-structure.md

```diff
@@ -0,0 +1,336 @@
+# Schema Public
+
+## Tables
+
+### audit
+
+Some exercises require reviews from other students or any other member of the school. The reviewer checks that some conditions are met by the code of the student being reviewed. This process is called an `audit` and it's kept in this table.
+
+Columns:
+- id.
+- groupId: The group that the audit refers to.
+- auditorId: The user who's making the audit.
+- attrs: The attributes of the audits keep the feedback of the auditor for each question failed.
+- grade: 0 for failed and 1 for passed.
```

Comment:
grade can actually be between 0 and 1 or more than one (it's relative to the number of required questions validated + they can have more if they succeeded bonus questions)
it also can be null (until the auditor do the audit)

Reply:
Ok, thank you for the info. I'll add it.